app/_includes/gateway-support-browsers.html

```diff
@@ -0,0 +1,4 @@
+{% for d in include.data.browsers %}
```

Comment:
I think we should figure out how to make the default selected tab the current version of the docs.  Otherwise we should default to the latest version (last in the list?).